docs/design-plans/2026-05-25-identity-risk-engine-spec.md

```
@@ -0,0 +1,312 @@
+| Email disposable       | +0.30                        |
+| Domain age < 30 days   | +0.25                        |
+| Domain age 30–180 days | +0.10                        |
+| No MX records          | +0.30                        |
+| Domain not registered  | +0.50 (floor verified=false) |
+| IP VPN/proxy/TOR       | +0.20                        |
+
+---
+
+## Go package structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)